src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -324,6 +324,34 @@ public void cannotCreateHoldRequestWhenItemIsNotCheckedOut()
     assertThat(postResponse, hasStatus(HTTP_VALIDATION_ERROR));
   }
 
+  @Test
+  public void cannotCreateRequestItemAlreadyCheckedOutToRequestor()
+    throws InterruptedException,
+    ExecutionException,
+    TimeoutException,
+    MalformedURLException {
+
+      final IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
```

Comment:
All of this method body has an extra level of indentation than is needed. Is there a reason for that?